### PR TITLE
Add Docker and fix generation

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

File: transit_to_gif.py [hunk 1]
```diff
@@ -122,7 +122,7 @@
     points = [int(i) for i in points if i]
     ways_all_nodes.extend(points)
     w["nodes_ref"] = points
-print("Chargement des ways terminé : {:d}".format(len(ways)))
+print("Chargement des ways termine : {:d}".format(len(ways)))
```
Comment (nlehuby) [inline transit_to_gif.py]: why did you remove the accents in the prints ?

File: requirements.txt [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+pillow==4.3.0
+folium==0.6.0
+olefile==0.44
+jinja2==2.9.6
+shapely==1.6.4.post2
+urllib3==1.23
+python-dateutil==2.6.1
+markupsafe==1.0
+imageio==2.2.0
+requests==2.20.0
```
Comment (nlehuby) [inline requirements.txt]: not needed anymore

File: requirements.txt [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+selenium==3.6.0
+pytz==2017.2
+numpy==1.15.3
+chardet==3.0.4
+pillow==4.3.0
+folium==0.6.0
+olefile==0.44
+jinja2==2.9.6
+shapely==1.6.4.post2
+urllib3==1.23
```
Comment (nlehuby) [inline requirements.txt]: is it needed ?

File: transit_to_gif.py (hunk 1 above)
Comment (prhod) [reply transit_to_gif.py]: I couldn't solve an ascii error conversion when using `print`: 
UnicodeEncodeError: 'ascii' codec can't encode character '\xe9' in position 27: ordinal not in range(128)
(checked it again this morning)

File: requirements.txt (hunk 2 above)
Comment (prhod) [reply requirements.txt]: right ! fixed

File: requirements.txt (hunk 3 above)
Comment (prhod) [reply requirements.txt]: fixed

File: transit_to_gif.py (hunk 1 above)
Comment (nlehuby) [reply transit_to_gif.py]: You can fix this in the `Dockerfile`:
* install `locales` ubuntu package
* generate the locale with `RUN locale-gen en_US.UTF-8`
* use it at default: `ENV LANG en_US.UTF-8`

see also: https://webkul.com/blog/setup-locale-python3/

Comment (prhod) [reply transit_to_gif.py]: thanks a lot, it works like a charm :+1:
